addons/mail/static/src/rtc/call_action_list.xml

```diff
@@ -62,43 +51,25 @@
-                        t-att-class="{
-                            'p-2': isSmall,
-                            'p-3 m-1': !isSmall,
-                        }"
-                        t-att-aria-label="screenSharingText"
-                        t-att-title="screenSharingText"
-                        t-on-click="() => this.rtc.toggleVideo('screen')">
-                        <div class="fa-stack">
-                            <i class="fa fa-desktop fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall, 'text-success': rtc.state.sendScreen }"/>
-                        </div>
-                    </button>
-                    <button t-if="!props.fullscreen.isActive" class="btn d-flex border-0 rounded-circle shadow-none opacity-100 opacity-75-hover"
-                        aria-label="Enter Full Screen"
-                        title="Enter Full Screen"
-                        t-att-class="{ 'p-2': isSmall, 'p-3 m-1': !isSmall }"
-                        t-on-click="props.fullscreen.enter"
-                    >
-                        <div class="fa-stack">
-                            <i class="fa fa-arrows-alt fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall }"/>
-                        </div>
-                    </button>
-                    <button t-if="props.fullscreen.isActive" class="btn d-flex border-0 rounded-circle shadow-none opacity-100 opacity-75-hover"
-                        aria-label="Exit Full Screen"
-                        title="Exit Full Screen"
-                        t-att-class="{ 'p-2': isSmall, 'p-3 m-1': !isSmall }"
-                        t-on-click="props.fullscreen.exit"
-                    >
-                        <div class="fa-stack">
-                            <i class="fa fa-compress fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall }"/>
-                        </div>
-                    </button>
+                    <Dropdown position="'top-end'" togglerClass="`btn d-flex m-1 border-0 rounded-circle
+                    shadow-none opacity-100 opacity-75-hover ${ isSmall ? 'p-2' : 'p-3' }`" menuClass="'d-flex flex-column py-0'">
+                        <t t-set-slot="toggler">
+                            <div class="fa-stack">
+                                <i class="fa fa-ellipsis-v fa-stack-1x" t-att-class="{ 'fa-lg': !isSmall }" title="More" aria-label="More"/>
+                            </div>
+                        </t>
+                        <t t-set-slot="default">
+                            <DropdownItem t-foreach="actions" t-as="action" t-key="action.id"
+                            class="'btn rounded-0 d-flex align-items-center px-2 py-2 m-0 opacity-75
+                            opacity-100-hover'" title="action.name" onSelected="action.onSelect">
+                                <i t-att-class="action.icon"/>
+                                <span class="mx-2" t-out="action.name" t-att-aria-label="action.name"/>
+                            </DropdownItem>
```

Comment:
Do not wrap line manually. In rare cases (e.g. `t-att-class` with object syntax) it's alright to do multi-line, most of the time we put everything on same line and let IDE wrap line (if desired).
XML sucks, but it's better for everyone to just avoid manual formatting, as the global shape of the template is more important than making the details very apparent (at the cost of having a hard time to see the template as a whole).